### PR TITLE
Timeouts endpoint to spec.

### PR DESCRIPTION
**Pull request by rafael-chavez**

### CODE REVIEW

**Comment by rafael-chavez** [comment]
@marekcirkos

File: WebDriverAgentLib/Commands/FBCustomCommands.m [hunk 1]
```diff
@@ -30,7 +30,7 @@ + (NSArray *)routes
   @[
     [[FBRoute POST:@"/homescreen"].withoutSession respondWithTarget:self action:@selector(handleHomescreenCommand:)],
     [[FBRoute POST:@"/deactivateApp"] respondWithTarget:self action:@selector(handleDeactivateAppCommand:)],
-    [[FBRoute POST:@"/timeouts/implicit_wait"] respondWithTarget:self action:@selector(handleImplicitWaitCommand:)],
```
Comment (marekcirkos) [inline WebDriverAgentLib/Commands/FBCustomCommands.m]: Could you not remove old endpoint and add TODO: comment to remove it. I need migrate your clients before I can remove it

Comment (rafael-chavez) [inline WebDriverAgentLib/Commands/FBCustomCommands.m]: Are you referring to this ? https://github.com/SeleniumHQ/selenium/commit/65dc17ed3435a0c4d1935e45314f4913f4e02d4e

This has already been refactored to be standard compliant on the java side. Not sure about the other clients (ruby, python, etc..)

Comment (marekcirkos) [inline WebDriverAgentLib/Commands/FBCustomCommands.m]: I mean our client, that we use for e2e testing :).

Comment (rafael-chavez) [inline WebDriverAgentLib/Commands/FBCustomCommands.m]: Updated @marekcirkos

**Comment by facebook-github-bot-9** [comment]
@rafael-chavez updated the pull request.

File: WebDriverAgentLib/Commands/FBCustomCommands.m (hunk 1 above)
Comment (marekcirkos) [inline WebDriverAgentLib/Commands/FBCustomCommands.m]: Thank you!

**Comment by marekcirkos** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-8** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1542272286079856/int_phab) to review.

**Comment by marekcirkos** [comment]
I think there is conflict in project file

**Comment by marekcirkos** [comment]
Omg, bot grabbed changes from your first diff as well, weird

**Comment by marekcirkos** [comment]
@facebook-github-bot import

**Comment by rafael-chavez** [comment]
hmmm thats odd.

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1542272286079856/int_phab) to review.

**Comment by marekcirkos** [comment]
Same thing, it is like it is based on old rev. Can you rebase?

**Comment by rafael-chavez** [comment]
you want me to close this pull and rebase my changes onto master?

**Comment by rafael-chavez** [comment]
Sorry not sure what you mean by rebase since there branch that contains the changes is the same as the branch that this is going into. Usually you rebase from a diverged branch into another branch.

**Comment by marekcirkos** [comment]
I am on holiday with limited internet access so it is hard for me diagnose what is wrong. It looks like our bot is importing our previous closed PR (with same name). If you could open new one with different name, I might be able to import it.

**Pull request closed without merging**